packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -54,9 +69,19 @@ function BlockGroupToolbar( { label = _x( 'Group', 'verb' ) } ) {
 		<ToolbarGroup>
 			<ToolbarButton
 				icon={ group }
-				label={ label }
+				label={ _x( 'Group', 'verb' ) }
 				onClick={ onConvertToGroup }
 			/>
+			<ToolbarButton
+				icon={ row }
+				label={ _x( 'Row', 'verb' ) }
```

Comment:
I just noticed this. We might want to provide better context for `Row`.

Reply:
Hah, indeed, this seems like it's implying it's the verb of rowing :)

Reply:
I've updated this to `label={ _x( 'Row', 'single horizontal line' ) }`. Can tweak further if needed.